frontend/src/index.mempool.html

```diff
@@ -1,10 +1,15 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>mempool - Bitcoin Explorer</title>
   <base href="/">
 
+  <script>
+    document.write("<script src=\"/resources/config.js?v=" + new Date().getTime() + "\"><\/script>");
+  </script>
+
```

Comment:
What is the intention of this ugly hack, to bypass cache or something?

Reply:
Yes - appending the timestamp so we always get a fresh config

Reply:
Instead of this hack, let's serve the config.* with no-cache and expires -1 in nginx.conf ?

Reply:
We can try that. 

Can you share the actual change needed to the nginx config so I can make the docker changes to apply the overrides and try on a real image?

Reply:
In the `production/nginx/server-common.conf` there is a block like this:
```
# cache /resources/** for 1 week since they don't change often
location /resources {
        try_files $uri /en-US/index.html;
        expires 1w;
}
```
below it I would add a more specific match block:
```
# only cache /resources/config.* for 10 seconds since it changes often
location /resources/config. {
        try_files $uri =404;
        expires 10s;
}
```

Reply:
Since the `index.html` is cached for 5 minutes, that seems like a reasonable time to cache the `config.js` files for as well

Reply:
You'll also need to modify the top-level `nginx-mempool.conf` with a similar block, since they are still separate until @erikarvstedt merges them in his PR for that later